### ISSUE TITLE
ObjLoader

### ISSUE THREAD

**Issue by glmcz** [ngokevin/aframe-react#102]
Hi all, I was traying to load obj model throught aframe-react component <Entity> whit code below, but without any success. Did I have to define geometry and material props ? If yes how could I get them from obj-loader ? 
  <Entity obj-model={{src: './src/h06_low.obj'}}>  

PS:  <Entity obj-model={{obj:  './src/h06_low.obj' }}/> too without success ...

Can someone please provide any tips or example how to load obj model ?

**Comment by AndresCuervo** [comment]
Components expect a string to parse, does:

```HTML
<Entity obj-model='obj: src/h06_low.obj' />
```

work for for you?

**Comment by glmcz** [comment]
Yes many times... , without success... Only plain HTML attributes

![no-obj-model](https://user-images.githubusercontent.com/18687535/30563878-872ba1d0-9cc3-11e7-8722-a48be47d9341.PNG)

**Comment by AndresCuervo** [comment]
Have you tried putting the model in the `public` directory instead of the `src` directory? I'm not sure how serving things in the `src` directory works, but you could try putting `h06_low.obj` in `public` and then do something like:

```html
<Entity obj-model='obj: h06_low.obj' />
```

That structure works for me

**Comment by AndresCuervo** [comment]
I got the `public` serving by working off of the [A-Frame React boilerplate](https://github.com/ngokevin/aframe-react-boilerplate), try cloning that and putting your model in `public` and using `obj-model` as I suggested above?

**Comment by glmcz** [comment]
Yes I`m using [A-Frame React boilerplate](https://github.com/ngokevin/aframe-react-boilerplate) without any modification,for the sake I cloned it again, but still none of this paths work for me. I can`t see object even if I use different obj model .... using Windows 7 and model was moved to public dir  ...
  <Entity obj-model='obj: has.obj' />
  <Entity obj-model='obj: ./public/cup.obj' />
